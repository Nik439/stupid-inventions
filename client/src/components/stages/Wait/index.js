import React from 'react';
import './styles.css';

function Wait() {
  return (
    <div className="wait-container">
      <h1 data-testid="h1" className="wait-text">
        Waiting for other players...
      </h1>
    </div>
  );
}

export default Wait;
