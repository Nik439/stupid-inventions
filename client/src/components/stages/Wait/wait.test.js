import {render, screen} from '@testing-library/react';
import Wait from '.';

describe('Wait', () => {
  test('should render waiting message', () => {
    render(<Wait />);

    expect(screen.getByTestId('h1')).toMatchSnapshot();
  });
});
