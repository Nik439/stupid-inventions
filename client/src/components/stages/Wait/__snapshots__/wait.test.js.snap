// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wait should render waiting message 1`] = `
<h1
  class="wait-text"
  data-testid="h1"
>
  Waiting for other players...
</h1>
`;
